examples/framework/preferences/shared/html/slider-template.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    <label class="mpe-slider-label"></label>
+    <div class="mpe-slider">
+        <input aria-labelledby="text-size-label" class="flc-textfieldSlider-slider fl-slider">
```

Comment:
Sorry, my above comment is wrong. `mpe-slider-label` is actually a css class on the label element at line 2. An `id` value should be added to that label element and be used as `aria-labelledby` values at line 4, 5.